nexong/api/routers.py

```diff
@@ -2,12 +2,23 @@
 from .Authentication.views import *
 from .Meeting.views import *
 from .Event.views import *
+from .Evaluation.views import *
 
 router_api = DefaultRouter()
 router_api.register(prefix="user", viewset=UserApiViewSet, basename="user")
 router_api.register(prefix="meeting", viewset=MeetingApiViewSet, basename="meeting")
 router_api.register(prefix="event", viewset=EventApiViewSet, basename="event")
-router_api.register(prefix="volunteer", viewset=EventApiViewSet, basename="volunteer")
 router_api.register(
     prefix="lesson-event", viewset=LessonEventApiViewSet, basename="lessonevent"
 )
+router_api.register(
+    prefix="student-evaluation",
+    viewset=StudentEvaluationApiViewSet,
+    basename="studentevaluation",
+)
+router_api.register(
+    prefix="evaluation-type",
+    viewset=EvaluationTypeApiViewSet,
+    basename="evaluationtype",
+)
+router_api.register(prefix="volunteer", viewset=EventApiViewSet, basename="volunteer")
```

Comment:
There is a mistake here but it has nothing to do with your task. I would delete this because it is related to another PR

Reply:
Line was erased as requested